schema/admins/locality.yaml

```diff
@@ -36,3 +36,6 @@ properties:
       contextId:
         description: Context entity is the most granular entity that logically contains given entity (but doesn't have to contain it spatially due to minor discrepancies in geometries)
         type: string
+      population:
+        description: Number of citizen in locality.

```

Comment:
nit: "Number of citizen**s**".

You could just say "population in the locality" or something like that too.

Reply:
Done.